jsonrpc/eth_endpoint.go

```
@@ -1087,3 +1089,89 @@ func (e *Eth) signTx(args *txnArgs) (*types.Transaction, error) {
 
 	return keyStore.SignTx(account, tx)
 }
+
+// GetProof returns the Merkle-proof for a given account and optionally some storage keys.
+func (e *Eth) GetProof(address types.Address, storageKeys []string, filter BlockNumberOrHash) (interface{}, error) {
+	header, err := GetHeaderFromBlockNumberOrHash(filter, e.store)
+	if err != nil {
+		return nil, err
+	}
+
+	snap, err := e.store.NewSnapshotAt(header.StateRoot)
+	if err != nil {
+		return nil, err
+	}
+
+	account, err := snap.GetAccount(address)
```

Should we return an error if there is no account found?